docs/book/src/reference/metrics.md

```diff
@@ -73,6 +76,29 @@ for the metrics exported from the namespace where the project is running
 
 <img width="1680" alt="Screenshot 2019-10-02 at 13 07 13" src="https://user-images.githubusercontent.com/7708031/66042888-a497da80-e515-11e9-9d77-d8a9fc1159a5.png">
 
+## Default Metrics References
+
+By default, [controller-runtime](https://github.com/kubernetes-sigs/controller-runtime) provides controller-relative metrics in different perspectives:
+
+| Metrics name                                  | Type      | Description                                                                                                                                                                                                                          |
+| --------------------------------------------- | :-------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| workqueue_depth                               | Gauge     | Current depth of workqueue.                                                                                                                                                                                                          |
+| workqueue_adds_total                          | Counter   | Total number of adds handled by workqueue.                                                                                                                                                                                           |
```

Comment:
Are these client-go metrics under the hood (https://pkg.go.dev/k8s.io/client-go/util/workqueue#MetricsProvider) or controller-runtime specific reconciler utilizes it? If its client-go related, would be helpful to drop a link so that its easier to know where they are used.

Reply:
Hey, @varshaprasad96 , thx for reviewing!
The workqueue related metrics listed here is refer to https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/metrics/workqueue.go
More specifically, [this code block](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/metrics/workqueue.go#L47) is where I got `workqueue_adds_total`.
So it seems like controller-runtime specific?